Add info about refreshing software repositories (#584)
uyuni-project/uyuni-docs: modules/upgrade/pages/server-minor-upgrade-uyuni.adoc

+
Reboot the server if a patch update recommends rebooting.

[NOTE]
====

> With default configuration setttings, all zypper commands run a repository refresh every 10 minutes (see ``repo.refresh.delay`` in [path]``/etc/zypp/zypp.conf``).

Reviewer: ```suggestion
By default, zypper refreshes the repository every ten minutes (see ``repo.refresh.delay`` in [path]``/etc/zypp/zypp.conf``).
```

> If you disabled ``autorefresh``, only an explicit [command]``zypper ref`` command will refresh all repositories.

Reviewer: ```suggestion
If ``autorefresh`` is disabled, run [command]``zypper ref`` to refresh all repositories.
```

====

> // We can remove  the version info when 4.1 / 2020.07 is released

Reviewer: these have been released?
